database/index.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require('mongoose');
+mongoose.connect('mongodb://localhost:27017/moodu');
+
+
+/* SCHEMAS */
+let reviewsSchema = mongoose.Schema({
+    id: 'number',
+    filmname: 'string',
+    reviews: [
+      {
+        id: 'number',
+        review: 'string',
+        author: 'string',
+        rating: 'number',
+        source: 'string'
+      }
+    ]
+  });
+
+
+let productsSchema = mongoose.Schema({

```

Comment:
Check your indentations! (ln 21-29)

Reply:
+1
Please set up your linter immediately so that your tooling can let you know about these issues (rather than having your PR reviewers try to catch all of these issues.)